docs/documentation.html

```diff
@@ -76,11 +76,11 @@
 .text_color
 .text_size
 .text_origin
-.fit_to_text()
+.fit_to_text() <gray># Creates background and fits text to it</gray>
 
-.on_click
-.on_enable
-.on_disable
+.on_click <gray># To check if button clicked</gray>
```

Comment:
This is not to check if a button has been clicked. It's to bind a function to be called when you click the button.

Reply:
ah right my apologies, i don't know how i messed that up